grapher/stackedCharts/StackedDiscreteBarChart.tsx

```diff
@@ -362,6 +379,209 @@ export class StackedDiscreteBarChart
         )
     }
 
+    private renderBar(bar: Bar, tooltipProps: TooltipProps) {
+        const { axis, formatColumn, focusSeriesName, barHeight } = this
+        const { point, color, seriesName } = bar
+
+        const isFaint =
+            focusSeriesName !== undefined && focusSeriesName !== seriesName
+        const barX = axis.place(this.x0 + point.valueOffset)
+        const barWidth = axis.place(point.value) - axis.place(this.x0)
+
+        // Compute how many decimal places we should show.
+        // Basically, this makes us show 2 significant digits, or no decimal places if the number
+        // is big enough already.
+        const dp = Math.ceil(-Math.log10(point.value) + 1)
+        const barLabel = formatColumn.formatValueShort(point.value, {
+            numDecimalPlaces: isFinite(dp) && dp >= 0 ? dp : 0,
+        })
+        const labelBounds = Bounds.forText(barLabel, {
+            fontSize: 12.6,
```

Comment:
Why is the `fontSize` hardcoded? Shouldn't it depend on `baseFontSize`?

Reply:
Good catch!